src/config.php

```diff
@@ -28,6 +28,6 @@
 return [
     'pluginNameOverride' => null,
     'templateOverride' => null,
-    'section' => null, // section ID (int) or handle (string)
```

Comment:
That is a possible breaking change, but in my tests, using a string for the section didn't work as expected with the default templates. It would throw an error when trying to view the User Manual. You'd be required to write a custom Twig template for it to work properly.